app.go

```diff
@@ -86,6 +86,8 @@ type AppInfo struct {
 	UserMetadataSchemaResponse    []UserMetadataField `json:"user_metadata_schema"`            // The schema for user_metadata that will be stored about users
 	Layouts                       Layouts             `json:"layouts"`                         // The layouts of user_metadata on the register/profile element
 	DefaultLanguage               string              `json:"default_language"`                // The default_language to be used by the app
+	AuthFallbackMethod            string              `json:"auth_fallback_method"`
+	AuthFallbackMethodTTL         int                 `json:"auth_fallback_method_ttl"`
```

Comment:
It might be worth adding the comment descriptions here too; it gets picked up by gopls as field documentation.
For example, in vim this looks like: 
<img width="297" alt="Screenshot 2023-04-05 at 4 32 06 PM" src="https://user-images.githubusercontent.com/13988480/230216432-0f5bc33d-ded6-4b61-b315-b8f94e93eced.png">